client/client-multi/client-java-core/src/main/java/org/kaaproject/kaa/client/KaaClient.java

```diff
@@ -55,7 +56,7 @@
      * @param record
      *            New log record object
      */
-    void addLogRecord(Log record);
+    BucketFuture addLogRecord(Log record);
```

Comment:
So a raw Future object should be returned or a template parameter should be specified as well.
Do we need to apply these changes to LogCollector.java?